docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -205,125 +181,141 @@ You should see the contents of the project's `data/` directory printed to your t
-
-### Configure the job
-
-Configure the job with the following settings:
-
-- Enter `iris-databricks` in the `Name` field.
-- In the dropdown menu for the `Type` field, select `Python wheel`.
-- In the `Package name` field, enter `iris_databricks`. This is the name of your package as defined in your project's `src/setup.py` file.
-- In the `Entry Point` field, enter `databricks_run`. This is the name of the [entry point](#create-an-entry-point-for-databricks) to run your package from.
-- Ensure the job cluster you created in step two is selected in the dropdown menu for the `Cluster` field.
-- In the `Dependent libraries` field, click `Add` and upload [your project's `.whl` file](#package-your-project), making sure that the radio buttons for `Upload` and `Python Whl` are selected for the `Library Source` and `Library Type` fields.
-- In the `Parameters` field, enter the following list of runtime options:
+### Create asset bundles files
+
+
+
+#### 1. Create the bundle
+
+Create a folder called `assets` in the root directory containing the file `assets/batch-inference-workflow-asset.yml`, this name is just an example:
+```yaml
+common_permissions: &permissions
+  permissions:
+    - level: CAN_VIEW
+      group_name: users
+
+experimental:
+  python_wheel_wrapper: true
+
+artifacts:
+  default:
+    type: whl
+    build: kedro package
+    path: ..
+resources:
+  jobs:
+    iris-databricks-job: #used to run job from CLI
+      name: "iris-databricks" #name of the job in databricks
+      tasks:
+        - task_key: kedro_run
+          existing_cluster_id: ${var.existing_cluster_id}
+          python_wheel_task:
+            package_name: iris #<package_name>
+            entry_point: databricks_run
+            named_parameters:
+              --conf-source: /Workspace${workspace.file_path}/conf
+              --package-name: iris #<package_name>
+              --env: ${var.environment}
+          libraries:
+            - whl: ../dist/*.whl
+      schedule:
+        quartz_cron_expression: "0 0 11 * * ?" # daily at 11am
+        timezone_id: UTC
+      <<: *permissions
+```
+This file provides a template for a job utilizing Kedro as a package, and it allows for multiple definitions per job. For instance, you could have one definition per pipeline.
+
+
+In this example, an existing cluster ID is being utilized, which can facilitate fast iteration. However, for a productive deployment, a job cluster is required. To enable ephemeral clusters, the YAML file (`assets/batch-inference-workflow-asset.yml`) needs to be modified to use a `job_cluster_key` instead of an `existing_cluster_id`.
+
+```yaml
+jobs:
+    iris-databricks-job: #used to run job from CLI
+      name: "iris-databricks"  #name of the job in databricks
+      job_clusters:
+        - job_cluster_key: job_cluster
+          new_cluster:
+              #Azure nodes
+              node_type_id: Standard_DS3_v2
+              spark_version: 14.3.x-scala2.12
```

Comment:
There are some issues when specifying a cluster with no workers, which was fixed in cli version `v.0.221.1`. I was able to create a cluster and run spark with no issues following the configuration from [here](https://github.com/databricks/bundle-examples/blob/main/dbt_sql/resources/dbt_sql_job.yml). See details of the problem [here](https://github.com/databricks/cli/issues/1472) and [here](https://community.databricks.com/t5/data-engineering/asset-bundles-cannot-run-job-with-single-node-job-cluster/td-p/72050).